src/AgDatabase.cs

```diff
@@ -131,9 +131,19 @@ public List<BackupMetadata> RecentBackups()
     {
       // find most recent full backup LSN across all replica servers
       var fullBackupLsnBag = new ConcurrentBag<decimal>();
-      _listener.ForEachAgInstance(s => fullBackupLsnBag.Add(s.Database(Name).MostRecentFullBackupLsn()));
+      _listener.ForEachAgInstance(s => 
+      {
+        try
+        {
+          fullBackupLsnBag.Add(s.Database(Name).MostRecentFullBackupLsn());
+        }
+        catch { }
+      });
 
       // find all backups in that chain
+      if (fullBackupLsnBag.Count == 0)
```

Comment:
Is there a `fullBackupLsnBag.IsEmpty()`?

Reply:
yup, looks like there is - changed it 👍🏽